lib/SyTest/Federation/Datastore.pm

```diff
@@ -200,6 +200,64 @@ sub create_event
    return $event;
 }
 
+=head2 get_backfill_events
+
+   @events = $store->get_backfill_events( start_at => \@ids, ... )
+
+Returns a list of events, starting from the event(s) whose ID(s) are given by
+the C<start_at> argument, and continuing backwards through their
+C<prev_events> linkage. They are returned in a linearized order, most recent
+first.
+
+The following other named arguments affect the behaviour:
+
+=over 4
+
+=item limit => $limit
+
+Gives the maximum number of events that should be returned.
+
+=item stop_before => \@ids
+
+Gives a list of event IDs that should not be entered into or returned (most
+likely because the caller already has them). These events do not count towards
```

Comment:
Yes. The intention here was more to suggest that the `start_at` list was included in the output, but the `stop_before` list was not.